app/views/test.scala.html

```diff
@@ -2223,12 +2226,38 @@
           Monthly</label></li><li class="form__radio-group-item"><input id="contributionType-ANNUAL" class="form__radio-group-input" type="radio" name="contributionType" value="ANNUAL" checked><label for="contributionType-ANNUAL" class="form__radio-group-label">
           Annual</label></li></ul></fieldset><fieldset class="form__radio-group form__radio-group--pills form__radio-group--contribution-amount"><legend class="form__legend form__legend--radio-group">
-          Amount</legend><ul class="form__radio-group-list"><li class="form__radio-group-item"><input id="contributionAmount-50" class="form__radio-group-input" type="radio" name="contributionAmount" value="50"><label for="contributionAmount-50" class="form__radio-group-label" aria-label="50 pounds">
-          @{annualAmounts._1}</label></li><li class="form__radio-group-item"><input id="contributionAmount-100" class="form__radio-group-input" type="radio" name="contributionAmount" value="100" checked><label for="contributionAmount-100" class="form__radio-group-label" aria-label="100 pounds">
-        @{annualAmounts._2}</label></li><li class="form__radio-group-item"><input id="contributionAmount-250" class="form__radio-group-input" type="radio" name="contributionAmount" value="250"><label for="contributionAmount-250" class="form__radio-group-label" aria-label="250 pounds">
-        @{annualAmounts._3}</label></li><li class="form__radio-group-item"><input id="contributionAmount-500" class="form__radio-group-input" type="radio" name="contributionAmount" value="500"><label for="contributionAmount-500" class="form__radio-group-label" aria-label="500 pounds">
-        @{annualAmounts._4}</label></li><li class="form__radio-group-item"><input id="contributionAmount-other" class="form__radio-group-input" type="radio" name="contributionAmount" value="other"><label for="contributionAmount-other" class="form__radio-group-label">
-          Other</label></li></ul></fieldset><div class="stripe-payment-request-button stripe-payment-request-button--hidden"></div><div class="form-fields"><div class="form__field form__field--contribution-email"><label class="form__label" for="contributionEmail">
+          Amount</legend><ul class="form__radio-group-list">
+          <li class="form__radio-group-item">
+            <span class="amounts__placeholder">
+            <input id="contributionAmount-50" class="form__radio-group-input" type="radio" name="contributionAmount" value="50">
+            <label for="contributionAmount-50" class="form__radio-group-label" aria-label="50 pounds">
+            </label></span>
+          </li>
+          <li class="form__radio-group-item">
+            <span class="amounts__placeholder">
+            <input id="contributionAmount-100" class="form__radio-group-input" type="radio" name="contributionAmount" value="100" checked>
```

Comment:
could we delete the inputs and `for` and `aria-label` attributes on the `<label>`? just to express the intentions more clearly - it took me a minute to realise that all of the numeric amounts were meaningless here

Reply:
for a11y reasons I'd put yhis entire preload body inside `aria-hidden` so that screen readers don't start reading the skeleton!

Reply:
good idea, didn't think about that!  maybe we should test with a screen reader!

Reply:
I left the inputs just for styling simplicity.  It started as a copy and paste of the html and css, but then we started tweaking the appearance a bit thus messing with the css and dom a bit, but I'm loathe to start refactoring too much as we are literally just doing it for the test result and reverting back at the end before we do next steps.  I definitely think we will implement it according to best practices if it produces a positive result.

The main thing is avoiding any kind of broken appearance to end users as that would affect the test result (e.g. confusing readers with screen readers or making the screen flash etc)